File: sdks/thunkmetrc-cs/ThunkMetrc.Wrapper/Models/CreatePlantsWasteRequest.cs
using System;
using System.Collections.Generic;

namespace ThunkMetrc.Wrapper.Models
{
    public class CreatePlantsWasteRequest
    {
        [global::System.Text.Json.Serialization.JsonPropertyName("LocationName")]
        public string? LocationName { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("MixedMaterial")]
        public string? MixedMaterial { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("Note")]
        public string? Note { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("PlantLabels")]
        public List<object>? PlantLabels { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("ReasonName")]
        public string? ReasonName { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("SublocationName")]
        public string? SublocationName { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("UnitOfMeasureName")]
        public string? UnitOfMeasureName { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("WasteDate")]
        public string? WasteDate { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("WasteMethodName")]
        public string? WasteMethodName { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("WasteWeight")]
        public double? WasteWeight { get; set; }
    }
}

File: sdks/thunkmetrc-java/wrapper/src/main/java/io/github/thunkier/thunkmetrc/wrapper/models/CreatePlantsWasteRequest.java
package io.github.thunkier.thunkmetrc.wrapper.models;import java.util.List;
import com.fasterxml.jackson.annotation.JsonProperty;public class CreatePlantsWasteRequest {
    @JsonProperty("LocationName")
    public String locationName;
    @JsonProperty("MixedMaterial")
    public String mixedMaterial;
    @JsonProperty("Note")
    public String note;
    @JsonProperty("PlantLabels")
    public List<Object> plantLabels;
    @JsonProperty("ReasonName")
    public String reasonName;
    @JsonProperty("SublocationName")
    public String sublocationName;
    @JsonProperty("UnitOfMeasureName")
    public String unitOfMeasureName;
    @JsonProperty("WasteDate")
    public String wasteDate;
    @JsonProperty("WasteMethodName")
    public String wasteMethodName;
    @JsonProperty("WasteWeight")
    public Double wasteWeight;
}
